src/c_sharp_graph/stack-graphs.tsg

```diff
@@ -107,6 +107,8 @@ attribute node_symbol = node            => symbol = (source-text node), source_n
   )
 ) @namespace {
     edge @namespace.def -> @class_declaration.def
+    edge @class_declaration.def -> @namespace.def
+    attr (@class_declaration.def -> @namespace.def) precedence = 10
```

Comment:
We need some way to document that precedence of 10 means that the edge is going back up to build the FQDN. 

The reason we have to do this right now is that the lexical information is not saved in the SQLite database, so we are unable to use it .